frontend/src/Login/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import {
+  View, StyleSheet, Text,
+  ImageBackground, Alert,
+} from 'react-native';
+import { useDispatch } from 'react-redux';
+import { useNavigation } from 'react-navigation-hooks';
+import { login } from './actionCreator';
+import Colors from '../common/colors';
+import background from '../../assets/login/background.jpg';
+import CardTextInput from './CardTextInput';
+import NextArrowButton from './NextArrowButton';
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 0,
+  },
+  header: {
+    width: 300,
+    marginVertical: 12,
+    fontSize: 24,
+    fontWeight: 'bold',
+    color: Colors.whiteColor,
+  },
+  background: {
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+});
+
+const mockedUser = {
+  username: 'kyya',
+  password: 'password',
+};
+
+function showAlert(text) {
+  Alert.alert('Tips', text);
```

Comment:
please rename from tips, I'm not sure what's the best practice for this, but I've never seen `Tips` in an application